src/main/java/uk/gov/hmcts/reform/coh/controller/validators/LinkedQuestionValidator.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.coh.controller.validators;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.coh.controller.question.QuestionRequest;
+import uk.gov.hmcts.reform.coh.domain.OnlineHearing;
+import uk.gov.hmcts.reform.coh.domain.Question;
+import uk.gov.hmcts.reform.coh.repository.QuestionRepository;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Service
+public class LinkedQuestionValidator implements BiValidator<OnlineHearing, QuestionRequest> {
+
+    @Autowired
+    private QuestionRepository questionRepository;
+
+    private Validation validation = new Validation();
+
+    private String message;
+
+    @Override
+    public boolean test(OnlineHearing onlineHearing, QuestionRequest questionRequest) {
+
```

Comment:
This bit is basically validating that the linked question id is for the same online hearing